rdr_service/resource/calculators/participant_ubr.py

```diff
@@ -53,8 +54,7 @@ def ubr_sexual_orientation(answer: (str, None)):
         # they did not answer the expected SexualOrientation_None response to the parent TheBasics_SexualOrientation
         # question first.  This is not  consistent with the documented survey branching logic.  Therefore, UBR/RBR
         # calculations are still based only on the response to the TheBasics_SexualOrientation question alone
-
-        if answer is None or answer == 'PMI_Skip':
+        if answer in [None, 'PMI_Skip', 'PMI_PreferNotToAnswer']:
             return UBRValueEnum.NotAnswer_Skip
 
         if answer not in ['SexualOrientation_Straight', 'PMI_PreferNotToAnswer']:
```

Comment:
Can we just change this to `if answer != 'SexualOrientation_Straight` now, for clarity?

Reply:
Yes, I'll make that change.